tachyon/crypto/commitments/fri/two_adic_fri_verifier.h

```diff
@@ -60,13 +60,12 @@ F VerifyQuery(uint32_t index, uint32_t log_max_num_rows,
   return folded_eval;
 }
 
-template <typename F, typename InputMMCS, typename ChallengeMMCS,
-          typename Challenger, typename OpenInput,
-          typename ExtF = typename ChallengeMMCS::Field>
+template <typename F, typename PCS, typename ChallengeMMCS, typename Challenger,
+          typename OpenInput, typename ExtF = typename ChallengeMMCS::Field>
 [[nodiscard]] bool TwoAdicFriPCSVerify(
     const TwoAdicFriConfig<ChallengeMMCS>& config,
-    const TwoAdicFriProof<ChallengeMMCS, std::vector<BatchOpening<InputMMCS>>,
-                          F>& proof,
+    const TwoAdicFriProof<ChallengeMMCS, std::vector<BatchOpening<PCS>>, F>&
+        proof,
```

Comment:
How about using `PCS::ChallengeMMCS` and `PCS::F`?

Reply:
Good!